vscode-extension/src/statusBar.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {StatusBarAlignment, StatusBarItem, window} from 'vscode';
+import {RelayExtensionContext} from './context';
+
+export function createStatusBar(): StatusBarItem {
+  const statusBar = window.createStatusBarItem(StatusBarAlignment.Right, 100);
+  statusBar.command = 'relay.showOutput';
+
+  return statusBar;
```

Comment:
Does this ever get disposed? If we make this a private property of a new `RelayStatusBar` class, we can do it there, if not, we probably need to do that wherever we clean up the context.